initialize.js

```diff
@@ -1,102 +1,117 @@
-  const contractIdsDir = `${dirname}/.soroban/contract-ids`;
-  const contractFiles = readdirSync(contractIdsDir);
+function contracts() {
+  const contractFiles = glob(`${dirname}/.soroban/contract-ids/*.json`);
 
-  contractFiles.forEach(contractFile => {
-    const contractPath = path.join(contractIdsDir, contractFile);
-    if (statSync(contractPath).size > 0) {  // Check if file is not empty
-      bind(contractPath);
-    }
-  });
+  return contractFiles
+    .map(path => ({
+      alias: filenameNoExtension(path),
+      ...JSON.parse(readFileSync(path))
+    }))
+    .filter(data => data.ids[process.env.SOROBAN_NETWORK_PASSPHRASE])
+    .map(data => ({alias: data.alias, id: data.ids[process.env.SOROBAN_NETWORK_PASSPHRASE]}));
 }
 
-function importContract(contract) {
-  const filenameNoExt = filenameNoExtension(contract);
-  const outputPath = `${dirname}/src/contracts/${filenameNoExt}.ts`;
-  const importContent = `import * as Client from '${filenameNoExt}';\n` +
-                        `import { rpcUrl } from './util';\n\n` +
-                        `export default new Client.Contract({\n` +
-                        `  ...Client.networks.standalone,\n` +
-                        `  rpcUrl,\n` +
-                        `});\n`;
-  
-  writeFileSync(outputPath, importContent);
-  console.log(`Created import for ${filenameNoExt}`);
+function bind({alias, id}) {
+  exe(`${cli} contract bindings typescript --contract-id ${id} --output-dir ${dirname}/packages/${alias} --overwrite`);
+}
+
+function bindAll() {
+  contracts().forEach(bind);
 }
 
-function import_all() {
-  const contractIdsDir = `${dirname}/.soroban/contract-ids`;
-  const contractFiles = readdirSync(contractIdsDir);
+function importContract({id, alias}) {
+  const outputDir = `${dirname}/src/contracts/`;
 
-  contractFiles.forEach(contractFile => {
-    const contractPath = path.join(contractIdsDir, contractFile);
-    if (statSync(contractPath).size > 0) {  // Check if file is not empty
-      importContract(contractPath);
-    }
-  });
+  mkdirSync(outputDir, { recursive: true });
+
+  const importContent =
+    `import * as Client from '${alias}';\n` +
+    `import { rpcUrl } from './util';\n\n` +
+    `export default new Client.Client({\n` +
+    `  ...Client.networks.${process.env.SOROBAN_NETWORK},\n` +
+    `  rpcUrl,\n` +
+    `${
+      process.env.SOROBAN_NETWORK === "local" || "standalone"
+        ? `  allowHttp: true,\n`
+        : null
+    }` +
+    `});\n`;
+
+  const outputPath = `${outputDir}/${alias}.ts`;
+
+  writeFileSync(outputPath, importContent);
+
+  console.log(`Created import for ${alias}`);
 }
 
-exe('echo $SOROBAN_ACCOUNT $SOROBAN_NETWORK_PASSPHRASE $SOROBAN_RPC_URL');
+function importAll() {
+  contracts().forEach(importContract);
+}
 
 // Calling the functions (equivalent to the last part of your bash script)
-fund_all();
-build_all();
-deploy_all();
-bind_all();
-import_all();
+fundAll();
+buildAll();
+deployAll();
+bindAll();
+importAll();
```

Comment:
We should delete this whole file, as `loam build` and `loam dev` should take care of it instead!

Reply:
I was surprised in your `README` that you said it works, but that's because of this!

I think for now, to use this with `soroban contract init`, we will need to manually edit the list of contracts and add back the `hello_world` one instead of `core` and `status_message`.

To make this actually work with `stellar contract init` we will need to implement the `cargo_workspace` functionality. Shit.

Reply:
Ah, I also think that the steps that work in the description of this PR are because I was running `npx loam build` without `LOAM_ENV`, so it was defaulting to production and the contracts are currently commented out in the production config in `environments.toml`.